docs/content/en/integrations/parsers.md

```diff
@@ -1292,3 +1292,7 @@ Import Yarn Audit scan report in JSON format. Use something like `yarn audit --j
 ### Zed Attack Proxy
 
 ZAP XML report format (with or without requests and responses).
+
+### Govulncheck 
```

Comment:
Hi. We are ordering the list of parsers alphabetically in the documentation. Can you move it a little bit higher, please?

Reply:
Hi!
No problem, I will do it